index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CSS Editor</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <div class=main>
+    <div>
+      <div class="search">
+        Please enter a site you would like to edit :) <br>
+        Or you can make changes to my portfolio! <br>
+        (Note: this doesn"t work for all websites)<br>
+        <input form="search" type="text" id="searchValue">
+        <input onClick="search()" form="search" type="submit" value="Submit">
+        <br>
+        (I couldn't get this to work for editing the iframe yet,<br>
+        so it currently edits the main page :P)
+      </div>
+      <div class="editor">
+        What would you like to change? <br>
+        Style Name: Value <br>
+        <form>
+          <input type="text" form="editor" id="style" name="editor">
+          <input type="text" form="editor" id="value" name="editor">
+          <input onClick="changeStyle()" form="changeStyle" type="submit" value="Submit">
+        </form>
+      </div>
+    </div>
```

Comment:
This is great. You provided a whole page that the user can edit and test different styles!

The downside is that there is no way we can provide our own HTML and style it.